blockchain/blockchain.go

```diff
@@ -661,6 +741,7 @@ func (bc *BlockChain) ResetWithGenesisBlock(genesis *types.Block) error {
 //
 // This method only rolls back the current block. The current header and current
 // fast block are left intact.
+// Deprecated: in order to repair chain, please use SetHead or setHeadBeyondRoot methods
```

Comment:
This function is deprecated, then are there any places to replace `repair` method to `setHeadBeyondRoot`?

Reply:
Lines 296, 309, 413 were supposed to use `repair`, but they are updated to `setHeadBeyondRoot`.